Source/GR32.Blend.SSE2.pas

```diff
@@ -1522,7 +1522,7 @@ procedure SIMD_4x0C080400;
   db $00, $04, $08, $0C
 end;
 
-procedure CombineMem_SSE41_Kadaif(F: TColor32; var B: TColor32; W: Cardinal); {$IFDEF FPC} assembler; nostackframe; {$ENDIF}
+procedure CombineMem_SSE41_Kadaif(F: TColor32; var B: TColor32; W: Cardinal); {$IFDEF FPC} assembler; {$IFDEF TARGET_X64}nostackframe;{$ENDIF} {$ENDIF}
```

Comment:
You are correct. I thought I added it because the compiler complained about something but I can't reproduce that anymore.

Unrelated to this, that particular function is broken and therefore disabled on FPC 64-bit.

**Expected** output (GradLines example):
![expected](https://github.com/graphics32/graphics32/assets/18141921/d23c095d-205b-46c6-b578-42325f841077)

**Actual**
![actual](https://github.com/graphics32/graphics32/assets/18141921/2db768ce-ddb7-46cc-ba82-d33bdc564908)

Reply:
Definitely broken, but so pretty  Hopefully we can get it resolved.

Reply:
Fixed